docs/user-guide/configuration/externalConfig.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: main
+title: External Config
+category: User Guide
+menu: menu
+toc:
+    - title: External Config
+      url: "#external-config"
+    - title: Configure external config in parent pipeline
+      url: "#configure-external-config-in-parent-pipeline"
+    - title: Parent and Child Relationship
+      url: "#parent-and-child-relationship"
+    - title: Parent and Child Relationship
+      url: "#parent-and-child-relationship"
+
+---
+# External Config
+External config allows a single parent pipeline to create and manage build configuration of other "child" pipelines with a single `screwdriver.yaml`. The source code for each child pipeline will be taken from each child repository.
+
+This feature will allow users who own multiple  source code repositories with the same workflow to manage their pipelines more easily.
+
+## Configure external config in parent pipeline
+In your parent repository's `screwdriver.yaml`, you can define child pipelines with the keyword `childPipelines`. Screwdriver will create or delete child pipelines on our behalf based on the listed `scmUrls`. Please make sure you have **admin** access in each child repository in order to manage child pipelines through this feature.
+
+```yaml
+childPipelines:
+   scmUrls:
+      - git@github.com:minz1027/test.template.git
+      - git@github.com:minz1027/quickstart-generic.git#master
+
+jobs:
+    main:
+        image: node:8
+        steps:
+            - install: npm install
+            - publish: npm publish
+```
+
+## Parent and Child Relationship
+| Pipeline      | Permissions   |
+| ------------- |:-------------:|
+| Parent     | All actions on its own pipeline plus create/delete/update/start child pipelines |
+| Child      | All actions on its own pipeline except delete/update itself; also, secrets are inherited from the parent pipeline by default but can be overwritten |
+
+## User Interface
+Parent pipeline UI:
+![External config parent](../assets/external-config.png)
```

Comment:
Technically this screenshot is wrong now, since @Filbird moved Child Pipelines to the first tab.

Reply:
That's not merged yet. will update it when it got merged